src/d3a/models/market/market_validators.py

```diff
@@ -0,0 +1,105 @@
+"""Validators for d3a markets."""
+
+from abc import ABC, abstractmethod
+from typing import Dict
+
+from d3a.models.market import Offer, Bid
+
+
+class Requirement(ABC):
+    """Define an offer/bid requirement with is_satisfied interface."""
+
+    @classmethod
+    @abstractmethod
+    def is_satisfied(cls, offer: Offer, bid: Bid, requirement: Dict) -> bool:
+        """Check whether a requirement is satisfied."""
```

Comment:
```suggestion
        """Check whether a requirement is satisfied."""
        pass
```
Correct me if I am wrong, as far as I know one needs to add `pass` for methods without any implementation.

Reply:
Adding docstring works as a place holder so no need for the pass statement